diagnostic.md

```diff
@@ -44,7 +53,10 @@ weather === "sunny"
 ```
 
 What do these expressions evaluate to?  Explain your answers.
-
+(weather = "sunny") = "sunny"
+(weather === "sunny") = true
+The first expression equates to sunny as we are assigning weather a string,
+the second one is true as it is comparing the two values and they are equal.
```

Comment:
You got these second two correct but you left out the first one.
`let weather` evaluates to `undefined`